src/platform/plugins/shared/workflows_management/server/plugin.ts

```diff
@@ -0,0 +1,288 @@
+                    {
+                      defaultMessage: 'Update',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    all: ['workflow'],
+                    read: [],
+                  },
+                  ui: ['updateWorkflow'],
+                },
+                {
+                  api: ['workflow:delete'],
+                  id: 'workflow_delete',
+                  name: i18n.translate(
+                    'platform.plugins.shared.workflows_management.featureRegistry.deleteWorkflowSubFeaturePrivilege',
+                    {
+                      defaultMessage: 'Delete',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    all: ['workflow'],
+                    read: [],
+                  },
+                  ui: ['deleteWorkflow'],
+                },
+                {
+                  api: ['workflow:execute'],
+                  id: 'workflow_execute',
+                  name: i18n.translate(
+                    'platform.plugins.shared.workflows_management.featureRegistry.executeWorkflowSubFeaturePrivilege',
+                    {
+                      defaultMessage: 'Execute',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    all: ['workflow_execution'],
+                    read: ['workflow'],
+                  },
+                  ui: ['executeWorkflow'],
+                },
+                {
+                  api: ['workflow:read'],
+                  id: 'workflow_read',
+                  name: i18n.translate(
+                    'platform.plugins.shared.workflows_management.featureRegistry.readWorkflowSubFeaturePrivilege',
+                    {
+                      defaultMessage: 'Read',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    read: ['workflow'],
+                    all: [],
+                  },
+                  ui: ['readWorkflow'],
+                },
+                {
+                  api: ['workflow_execution:read'],
+                  id: 'workflow_execution_read',
+                  name: i18n.translate(
+                    'platform.plugins.shared.workflows_management.featureRegistry.readWorkflowExecutionSubFeaturePrivilege',
+                    {
+                      defaultMessage: 'Read Workflow Execution',
+                    }
+                  ),
+                  includeIn: 'all',
+                  savedObject: {
+                    read: ['workflow_execution'],
+                    all: [],
+                  },
+                  ui: ['readWorkflowExecution'],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    });
```

Comment:
Can we extract this from the `/server/plugin.ts`? Maybe into a `/server/features.ts`

Reply:
@VladimirFilonov